charts/ratify/values.yaml

```diff
@@ -108,4 +110,5 @@ crds:
 
 # See https://github.com/deislabs/ratify/blob/main/docs/reference/usage.md for a list of available feature flags
 featureFlags:
-  # RATIFY_FEATURE_NAME: true
\ No newline at end of file
+  # RATIFY_FEATURE_NAME: true
+  RATIFY_CERT_ROTATION: true
```

Comment:
do we already have docs thats describe the values in ratify config?

Reply:
yes, there is a doc describing feature flags, I'll add new feature flag to this doc as well.